css-animations-2/Overview.bs

```diff
@@ -259,6 +259,8 @@ and the position of the animation in |element|'s 'animation-name' list,
      repeating the list as necessary as described in
      [[CSS-ANIMATIONS-1#animation-name]].
 
+1.   Let |default composite| be ''replace''.
```

Comment:
Should this default to the resolved value of 'animation-composite' for element in the same way we do for the timing function?

Reply:
It looks like we we have a test for that behaviour: https://github.com/web-platform-tests/wpt/blob/b5ddd150f64c457a28411545fa6c16685a33bf70/css/css-animations/KeyframeEffect-getKeyframes.tentative.html#L314-L331

Reply:
I thought that `animation-composite` should set the effect-wide `composite` operation rather than set a default for keyframes. I think the current behavior for `animation-timing-function` is a mistake since it doesn't let you set an effect-wide easing, and I'd rather we didn't repeat the same mistake.

Reply:
Oh, right you are. I completely forgot we had an effect-wide composite operation in the model.

I agree `animation-timing-function`'s behaviour is undesirable. We should find a way to fix that some time (perhaps introducing `animation-effect-timing-function`?).

I suppose even if we were to introduce `animation-effect-timing-function` the way the effect easing applies is different to the way the default composite operation applies so we don't need to worry about making this consistent with that.

Reply:
Yes, `animation-timing-function` has always bugged me. I filed https://github.com/w3c/csswg-drafts/issues/6982 so that we may discuss how to solve this mess.